R/chol_solve_linear_system.R

```diff
@@ -0,0 +1,7 @@
+# solves Ax = b for x
+chol_solve_linear_system <- function(A, b) {
+
```

Comment:
don't need all this extra whitespace, use an auto formatter. that will also make sure you're consistent with spacing between operators later, e.g. `1 + 2 * 4` and not `1+2 * 4`.